_posts/02-01-01-Code-Style-Guide.md

```diff
@@ -2,60 +2,43 @@
 anchor: code_style_guide
 ---
 
-# Code Style Guide {#code_style_guide_title}
+# কোড স্টাইল গাইড {#code_style_guide_title}
 
-The PHP community is large and diverse, composed of innumerable libraries, frameworks, and components. It is common for
-PHP developers to choose several of these and combine them into a single project. It is important that PHP code adhere
-(as close as possible) to a common code style to make it easy for developers to mix and match various libraries for
-their projects.
+পিএইচপি একটি বিশাল ও ভিন্ন ধরণের কমিউনিটি এবং এটি অসংখ্য লাইব্রেরি, ফ্রেমওয়ার্ক এবং অন্যান্য কম্পোনেন্টস দ্বারা গঠিত। পিএইচপি ডেভেলপাররা সাধারণত এইগুলোর মধ্যে কয়েকটি নির্বাচন করেন এবং সেগুলো একত্রিত করে একটি প্রজেক্টে ব্যবহার করেন। PHP কোডিং এর ক্ষেত্রে একটি সাধারণ কোড স্টাইল মেনে চলা (যতটুকু সম্ভব) খুবই গুরুত্বপূর্ণ যেন ডেভেলপারেরা তাদের প্রজেক্ট এর জন্য সহজেই বিভিন্ন লাইব্রেরি একত্রিত করে ও সাদৃশ্য বিবেচনা করে কাজ করতে পারেন।
```

Comment:
১২০ ক্যারেকটার পরে নতুন লাইনে লেখা দরকার

Reply:
The dilemma is:
- On one side, if we use line breaks within 120 characters, markdown will not treat them as a newline (unless blank line or 2 spaces at line ending). So we are safe to add line breaks to improve readability.
- On the opposite, if we keep 1 line of source content to 1 line of translated content, comparing and validating in PR would be easier.
 
At this point, my vote is for 1st option. I think the loss of not keeping line numbers similar to the source is ignorable compared to improved readability and adhering to best practices.